src/applecider/models/HyraxBaselineCLS.py

```diff
@@ -95,7 +94,7 @@ def train_step(self, batch):
             Dictionary containing the loss value for the current batch.
         """
 
-        labels = batch[1]
+        _, _, labels = batch
```

Comment:
same q as above, should this be `_, labels, _ = batch` ?

Reply:
Same comment as above :) I adjusted the order to help out with onnx-ification. Hopefully in the long run, the ordering won't matter, but it seems to make a difference for now because the to_onnx script will iterate through the tuple of inputs, taking particular actions for those that have data vs. those that are empty. And in this case, during inference, there was an empty array (`labels`) that was confusing the onnx exporter. Moving labels to be the last array in the list resolved the confusion.